config/clusters/leap/common.values.yaml

```diff
@@ -284,7 +284,8 @@ basehub:
             environment:
               NVIDIA_DRIVER_CAPABILITIES: compute,utility
             node_selector:
-              cloud.google.com/gke-nodepool: nb-gpu-t4
+              node.kubernetes.io/instance-type: n1-standard-8
+              cloud.google.com/gke-accelerator: nvidia-tesla-t4
```

Comment:
For GPU nodes, you can pin like this - but this level of pinning is only relevant if either we have a mix of GPUs or a mix of instance sizes with GPUs.

If we only have a single GPU node pool, no node selector is required at all because k8s gets that it needs to use a GPU node pool.